debian/control

```diff
@@ -0,0 +1,15 @@
+Source: qml-box2d
+Section: libs
+Priority: extra
+Maintainer: Ville Ranki <ville.ranki@iki.fi>
+Build-Depends: cdbs, debhelper (>= 7.0.50~), libbox2d-dev
+Standards-Version: 3.9.6
+
+Package: qml-box2d
+Architecture: any
+Depends: ${shlibs:Depends}, ${misc:Depends}, libbox2d2.3.0
```

Comment:
I see you have `Depends` and `Build-Depends` on libbox2d, but in fact `qml-box2d` ships with its own copy of Box2D which is built into the plugin, so it does not currently build against a system version of this library. So you should probably drop those dependencies.